package/android/build.gradle

```diff
@@ -278,6 +278,7 @@ if (ENABLE_PREFAB) {
         into "${project.buildDir}/headers/rnskia/"
         includeEmptyDirs = false
         include "**/*.h"
+        duplicatesStrategy = 'include'
```

Comment:
https://skia.googlesource.com/skia/+/refs/heads/main/RELEASE_NOTES.md#milestone-114 (the duplicated files link directly to the moved files and will be removed eventually from Skia).